### DISCUSSION TITLE
Impressões fixas ou checklist na entrada da OS

### DISCUSSION THREAD

**Discussion by SuperPorte** [RamonSilva20/mapos#1621] [Ideas]
Boa tarde galera!!
No meu sistema antigo, eu tinha uma opção de TODAS as OSs terem uma observação padrão na hora de imprimir. Segue:

> **!!!ATENÇÃO!!!**
> NÃO ENTREGAREMOS O EQUIPAMENTO SEM O CANHOTO DA ORDEM DE SERVIÇO OU UMA COMPROVAÇÃO DE POSSE E DOCUMENTO DE IDENTIFICAÇÃO COM FOTO
> _ _ _
> Situação de recebimento:
> Ligando: (_)Não (_)Sim, SEM vídeo (_) Sim COM vídeo e SEM sistema (_) Sim, COM vídeo e COM sistema
> Com Fonte: (_)Não (_)Sim / Com bateria: (_)Não (_)Sim
> _ _ _ 
> Serviços de HARDWARE (produtos, componentes, etc.) têm 90 dias de garantia.
> Serviços de SOFTWARE, ou seja, serviços de SISTEMA (programas, limpeza, recuperação de dados, etc.), NÃO TÊM GARANTIA!!
> Os SERVIÇOS são executados e CONFERIDOS no ATO de ENTREGA do equipamento, NÃO existindo garantias APÓS a entrega, pois o correto funcionamento do SISTEMA DEPENDE do USUÁRIO.
> Avisamos também que, equipamentos que estiverem AGUARDANDO CLIENTE, por mais de TRINTA dias (+ de 30 dias) CORRIDOS, será cobrada uma taxa de ESTADIA de CEM REAIS (R$100). A partir de então, a CADA 10 dias a taxa será ACRESCIDA de TRINTA REAIS (R$30).
> Caso o valor da ESTADIA ultrapasse o valor do CONSERTO (peças + serviços) OU do EQUIPAMENTO ABANDONADO, o equipamento do CLIENTE será LEILOADO para pagamento da DÍVIDA e liberação da vaga.
> 
> _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
> Consentimento e concordância do cliente.

Todas as minhas OSs saem com esse texto fixo, mas no MAPOS, tenho que ficar copiando e colando a cada nova OS. Onde em posso deixar isso padronizado?

Obrigado a todos!!

**Comment by guskid** [comment]
Fiz algo parecido. 
Coloquei no lugar onde eu queria que aparecesse um texto fixo dentro do arquivo "imprimirOs.php"  e "editarOs".
O código acrescentei foi:
`<table class="table table-bordered table-condensed">
                            <tbody>
                                <tr>
                                    <td><b>Importante saber:</b><br>
• TEXTO 1;<br>
•TEXTO 2;<br>
                                    <br>                                        
                                    </td>                                    
                                </tr>
                            </tbody>
                        </table>`
                        
Para mim ta resolvendo (y)

**Reply by guskid** [reply]
Só naqueles X ali que tem que achar outra solução.

**Reply by SuperPorte** [reply]
Pois é, essa é a questão. O povo aqui acha brecha para tudo. Se eu não imprimir o que marquei, vão falar que eu alterei a OS....
